proto/ethermint/evm/v1/evm.proto

```diff
@@ -240,3 +240,43 @@ message TraceConfig {
   // tracer_json_config configures the tracer using a JSON string
   string tracer_json_config = 13 [(gogoproto.jsontag) = "tracerConfig"];
 }
+
+// VirtualFrontierContract are virtual contracts that able to call directly without actual invoking EVM.
+// Prohibited to call directly from EVM.
+message VirtualFrontierContract {
```

Comment:
I'd suggest moving to separate, dedicated files for `vfc`

Reply:
done